azure_functions_worker/utils/common.py

```diff
@@ -11,14 +11,14 @@ def is_true_like(setting: str) -> bool:
     if setting is None:
         return False
 
-    return setting.lower().strip() in ['1', 'true', 't', 'yes', 'y']
+    return setting.lower().strip() in {'1', 'true', 't', 'yes', 'y'}
```

Comment:
why is this in a dictionary?

Reply:
This is a set now, rather than a list - faster lookup.